### PR TITLE
Add muerte-dragon.md to dragon.md

### PR DESCRIPTION
**Pull request by juliojesusvizcaino**
Please, add my contribution to the story @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2407)

### CODE REVIEW

File: espanol/dragon/muerte-dragon.md [hunk 1]
```diff
@@ -0,0 +1 @@
+La muerte del drag�n provoc� la ira de sus hermanos, que no dudaron en jurar venganza por aquel atroz acto.
```
Comment (casey-collab) [inline espanol/dragon/muerte-dragon.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by julioteleco** [comment]
Please, add my contribution to the story @casey-collab

File: espanol/dragon/muerte-dragon.md (hunk 1 above)
Comment (casey-collab) [inline espanol/dragon/muerte-dragon.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/dragon/muerte-dragon/muerte-dragon.md [hunk 2]
```diff
@@ -0,0 +1 @@
+La muerte del drag�n provoc� la ira de sus hermanos, que no dudaron en jurar venganza por aquel atroz acto.
```
Comment (casey-collab) [inline espanol/dragon/muerte-dragon/muerte-dragon.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**